app/src/main/java/ch/epfl/sdp/market/MarketActivity.java

```diff
@@ -124,7 +124,8 @@ private void syncMarketBackendToFrontend() {
         if (backend == null) finish();
         setupItemToViewMap();
         for (Item i : backend.getStock().keySet()) {
-            ((TextView) findViewById(itemToViewMap.get(i.getClass().hashCode()))).setText("Cost: " + backend.getStock().get(i).second + "\n" + "Value: " + (((int) (100 * i.getValue())) / 100.0));
+            String newText = "Cost: " + backend.getStock().get(i).second + "\n" + "Value: " + (((int) (100 * i.getValue())) / 100.0);
```

Comment:
is it not better to shorten this string?

Reply:
Same comment (this branch was merged into the other one)